Source/Core/Core/HW/EXI/BBA/XLINK_KAI_BBA.cpp

```diff
@@ -118,11 +118,12 @@ bool CEXIETHERNET::XLinkNetworkInterface::SendFrame(const u8* frame, u32 size)
   size += 4;
 
   // Only uncomment for debugging, the performance hit is too big otherwise
-  // INFO_LOG(SP1, "SendFrame %x\n%s", size, ArrayToString(m_out_frame, size, 0x10).c_str());
+  // INFO_LOG_FMT(SP1, "SendFrame {:x}\n{}", size, ArrayToString(m_out_frame, size, 0x10)));
```

Comment:
Technically, the same remark regarding hex specifier also applies here; but since its commented out it doesn't matter too much.